DISCLAIMER.md

```diff
@@ -0,0 +1,41 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Copyrights & Trademarks](#copyrights--trademarks)
+* [Guidelines](#guidelines)
+* [Update of Brands](#update-of-brands)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyrights & Trademarks
+
+The Simple Icons project contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file, in our NPM package and through links on our [website].
+
+Some icons are also in the project under specific `licenses`, and we will endeavour to include the relevant information in the JSON file. The lack of a license entry in the JSON does not imply there is no license, simply that we haven't catalogued it yet.
+
+We ask that users of icons in the collection read the guidelines and license agreements fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
+
+If a logo or icon includes a registered trademark or trademark symbol we follow the recommendations outline [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
+
+~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~
+
+## Guidelines
+
+> NOTE: the addition of guidelines to our project is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist.
+
+As guidelines are subject to change and, as we rely largely on our community to help us keep our project updated, the guidelines included in our project for a particular icon may be outdated. Therefore we also ask that users of our project do their own due diligence in researching a brand's guidelines. Additionally, we ask you to inform us of outdated brand guidelines through the [GitHub repository].
+
+## Update of Brands
+
+If the icon in the project does not match the current icon for a brand, please [submit an issue](https://github.com/simple-icons/simple-icons/issues/new/choose), and a member of the community will update the brand in due course. 
```

Comment:
I think we can make this more consistent with the "Removal of Brands" section, e.g.:

```suggestion
Should a brand wish for their icon or data to be updated, please [submit an issue] to the Simple Icons GitHub repository. In the issue, please explain your affiliation with the company and reasons for the update. We generally release updates once a week, but can occasionally make exceptions for immediate updates of brands. 
```